docs/conceptual-framework.md

```diff
@@ -437,3 +437,4 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
 + Results reproduced by [@royary](https://github.com/royary) on 2025-10-27 (commit [`d9d1a48`](https://github.com/castorini/pyserini/commit/d9d1a48d2437c1c6d6aa875ea468162d62501efc))
 + Results reproduced by [@Raptors65](https://github.com/Raptors65) on 2025-10-28 (commit [`8237181`](https://github.com/castorini/pyserini/commit/8237181239312494b2acaf514856598098c9923d))
 + Results reproduced by [@MahdiNoori2003](https://github.com/MahdiNoori2003) on 2025-10-29 (commit [`dc1ae1b`](https://github.com/castorini/pyserini/commit/dc1ae1be36dc924645a4ed03e3141ed0451b8415))
++ Results reproduced by [yektakia](https://github.com/yektakia) on 2025-10-31 (commit [`6d54799`](https://github.com/yektakia/pyserini/commit/6d547996acacb248c20b1b3e1a9f3d57aa2b1f79))
```

Comment:
This does not appear to be a valid URL:

https://github.com/yektakia/pyserini/commit/6d547996acacb248c20b1b3e1a9f3d57aa2b1f79